lib/car.rb

```diff
@@ -0,0 +1,33 @@
+class Car
+  attr_reader :make,
+              :model,
+              :monthly_payment,
+              :loan_length
+              :color
+
+  def initialize(make_model, monthly_payment, loan_length)
+    @make = "Ford"
+    @model = "Mustang"
```

Comment:
We don't want to hard code these values in our initialize. Rather we want to set them via the data provided to us from the `make_model` variable:

@make = make_model.split.first
@model = make_model.split.last